uint/src/uint.rs

```diff
@@ -445,7 +445,7 @@ macro_rules! construct_uint {
 		/// Little-endian large integer type
 		#[repr(C)]
 		$(#[$attr])*
-		#[derive(Copy, Clone)]
+		#[derive(Copy, Clone, Eq, PartialEq, Hash)]
```

Comment:
so, this won't work with length bigger than 32, correct? Is that a problem?

Reply:
correct, #388 is still unresolved, we'd need to figure out how to refactor our use of `unroll!` or the macro itself in order to fix that besides the 32-length limitation

But the 32-length limitation could be lifted soon with the stabilization of (subset of) const generics.

Reply:
https://github.com/rust-lang/rust/pull/74060

Reply:
ok, I see